parsons/google/google_bigquery.py

```diff
@@ -1136,6 +1109,46 @@ def get_row_count(self, schema: str, table_name: str) -> int:
 
         return result["row_count"][0]
 
+    def get_table_ref(self, table_name):
+        # Helper function to build a TableReference for our table
+        parsed = parse_table_name(table_name)
+        dataset_ref = self.client.dataset(parsed["dataset"])
+        return dataset_ref.table(parsed["table"])
+
+    def _get_job_config_schema(
+        self,
+        job_config: LoadJobConfig,
+        destination_table_name: str,
+        if_exists: str,
+        parsons_table: Optional[Table] = None,
+        custom_schema: Optional[list] = None,
+        template_table: Optional[str] = None,
+    ) -> LoadJobConfig:
```

Comment:
I think that the return type of this function is wrong. It looks like it should return an `Optional[List[SchemaField]]`, not a `LoadJobConfig` instance.